### PR TITLE
add FAQ page

### PR DESCRIPTION
**Pull request by francesmatthew**

### CODE REVIEW

File: react_source/components/FAQ.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        marginTop: '10vh',
+        marginBottom: '10vh'
+    },
+    sectionButton: {
+        width: '2em',
+        height: '2em',
+        fontSize: '1em',
+        alignItems: 'center',
+        justifyContent: 'center',
+        marginTop: '1.25em'
```
Comment (RyderReed15) [inline react_source/components/FAQ.js]:
```suggestion
```
Remove this to fix the error with the chevrons not aligning

**Review by RyderReed15 — APPROVED** [review]
Looks nice, maybe we should consider moving the questions and answers text to a separate file or something like that

**Comment by francesmatthew** [comment]
@RyderReed15 I changed the questions to be on a static JSON file that is fetched dynamically when the page is loaded. Is this a better solution?

**Comment by RyderReed15** [comment]
@francesmatthew Most likely, but not sure. we can discuss it at the meeting tomorrow. We could also explore statically rendering that page as an HTML file.

**Pull request merged**